ingest/importer/conversion/conversion_strategy.py

```diff
@@ -125,8 +125,9 @@ def __init__(self, field, main_category):
         self.main_category = main_category
 
     def apply(self, metadata: MetadataEntity, cell_data):
-        link_ids = self.converter.convert(cell_data)
-        metadata.add_external_links(self.main_category, link_ids)
+        if cell_data is not None and cell_data != '':
```

Comment:
I think `if cell_data:` will work similarly here as both `None` and `''` have `False` truthiness.

Reply:
Makes sense! Thanks!